### ISSUE TITLE
Double Clicking causes crash

### ISSUE THREAD

**Issue by sup3ruser** [johanneszab/TumblThree#66]
When double clicking  on personal notes/tags, it causes crash (on v.42)  Only way to not crash is you have to click twice very slowly, to give it enough time to recognize that you are wanting to input text.

Jerome

**Comment by tringate** [comment]
This problem has been around for many releases.  Actually it works fine if you do not move the mouse which is near impossible when you double click.  I discovered the movement issue when I used my notebook and it has the pad rather than a mouse and then the clickers so it is easy to not touch the pad and do 2 clicks.
I first noticed this when I attempted to delete the personal notes I had made. it constantly would crash. because I was trying to select just part of the note to delete.  You know, hold down the button and drag over what you want to delete to highlight it.
